persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraProvider.scala

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.actor.ActorSystem
+
+/**
+ * Internal API
+ */
+private[lagom] class CassandraProvider(system: ActorSystem) {
```

Comment:
I first call it CassandraConfigProvider, but I've found there is one already in "persistence-cassandra-javadsl" and I suppose that one should be moved later, but there are some dependencies, so I would like to avoid pushing so many changes as I am not that familiar with code base for now. Finally I've found SlickProvide with similar logic and just followed similar name convention even though I don't really like it :D Maybe CassandraReadSideConfigProvider, but it looks a bit long. Any suggestion?